src/renderer/views/Watch/Watch.js

```diff
@@ -1233,6 +1246,18 @@ export default defineComponent({
         return
       }
 
+      if (this.blockVideoAutoplay) {
+        showToast(this.$t('Autoplay Interruption Timer',
+          this.defaultAutoplayInterruptionIntervalHours,
+          {
+            autoplayInterruptionIntervalHours: this.defaultAutoplayInterruptionIntervalHours
+          }),
+        3_600_000
+        )
```

Comment:
I am surprised that linter is ok with this o_0

Reply:
It not only tolerates it: it enforces it to be like that. ¯\\_(ツ)_/¯

Reply:
The reason that is happening is because `showToast(` and `this.$t(` are both on the same line, so the linter sees `this.$t` as having one level of indentation and `showToast` as having none. That is why it forcing `3_600_000` and `)` to have no indentation.

Moving `this.$t` to its own line fixes the issue, as then the linter allows them both to have indentation:
![screenshot-of-fixed-formatting](https://github.com/user-attachments/assets/905a81a7-ae14-439f-ac31-2a6258d92873)

It would also improve the general readability of that code snippet if the placeholder name in the translation string weren't so long e.g. `{hours}` would already suffice, as there is only one placeholder in that string and there is enough context in the string and the code where it is used, to understand what the purpose of the placeholder is.

However as both things are stylistic issues and don't influence the functionality of the code, we can leave the cleanup for some future pull request.